_posts/en/newsletters/2025-04-04-newsletter.md

```diff
@@ -0,0 +1,569 @@
+      replacing their current need to perform Script-based lamport
+      signatures.  CTV many be able to reduce some additional signature
+      operations.  Poinsot again [asks][poinsot ctvcsfs1] about whether
+      there's significant demand for BitVM.  Gregory Sanders
+      [replies][sanders bitvm] that he would find it interesting for
+      bidirectional bridging of tokens as part of Shielded [client-side
+      validation][topic client-side validation] (see [Newsletter
+      #322][news322 csv-bitvm]).  However, he also notes that neither
+      CTV or CSFS significantly improve the trust model of BitVM,
+      whereas other changes might be able to reduce trust or reduce the
+      number of expensive operations in other ways.
+
+    - *Improvement in Liquid timelock script:* James O'Beirne
+      [relayed][obeirne liquid] comments from two Blockstream engineers
+      that CTV would, in his words, "drastically improve the
+      [Blockstream] Liquid timelock-fallback script that requires coins
+      to be [moved] on a periodic basis."  After requests for
+      clarification, former Blockstream engineer Sanket Kanjalkar
+      [explained][kanjalkar liquid] that the benefit could be a
+      significant reduction in fees.  O'Beirne also [shared][poelstra
+      liquid] additional details from Andrew Poelstra, Blockstream's
+      Director of Research.
+
+    - *LN-Symmetry:* CTV and CSFS together can be used to implement a
+      form of [LN-Symmetry][topic eltoo], which eliminates some of the
+      downsides of [LN-Penalty][topic ln-penalty] channels currently
+      used in LN and may allow the creation of channels with more than
+      two parties, which might improve liquidity management and onchain
+      efficiency.  Gregory Sanders, who implemented an experimental
+      version of LN-Symmetry (see [Newsletter #284][news284 lnsym])
+      using [SIGHASH_ANYPREVOUT][topic sighash_anyprevout] (APO)
+      [notes][sanders versus] that the CTV+CSFS version of LN-Symmetry
+      isn't as featureful as the APO version and requires making
+      tradeoffs.  Anthony Towns [adds][towns nonrepo] that nobody is
+      known to have updated Sanders's experimental code for APO to run
+      on modern software and use recently introduced relay features such
+      as [TRUC][topic v3 transaction relay] and [ephemeral
+      anchors][topic ephemeral anchors]; much less has anyone ported the
+      code to use CTV+CSFS, limiting our ability to evaluate that
+      combination for LN-Symmetry.
+
+      Poinsot [asks][poinsot ctvcsfs1] whether implementing LN-Symmetry
+      would actually be a priority for LN developers if a soft fork
+      making it possible was activated.  Quotes from two Core Lightning
+      developers (also co-authors of the paper introducing what we now
+      call LN-Symmetry) indicated that it was a priority for them.  A
+      quote from an LDK developer did not show it to be a priority.
```

Comment:
Did it not show that it was a priority or did it show not to be a priority?

```suggestion
      quote from an LDK developer showed it not to be a priority.
```

Reply:
Grr, I think I need to stop reporting on anything contained in a Twitter thread.  When I read @ajtowns's [link](https://x.com/TheBlueMatt/status/1859070516956389704), I only saw a subset of the conversation and AJ's conclusion seemed only weakly supported (at best).  I just checked again and it's clearly showing me an upthread post with Matt saying "I don't find eltoo all that interesting in terms of 'we need to get this done'".

I'll revise to use stronger language; thanks for double checking.

Reply:
Yeah, Matt has been of the position that Lightning needs a penalty for some time, so he is not particularly interested in LN Symmetry